.circleci/config.yml

```diff
@@ -11,6 +11,7 @@ jobs:
       - DOCKER_VERSION: 17.05.0-ce
       - DOCKER_COMPOSE_VERSION: 1.15.0
       - METEOR_ALLOW_SUPERUSER: true
+      - TOOL_NODE_FLAGS: "--max-old-space-size=4096"
```

Comment:
any further reference on the memory issue / fix here?

Reply:
There is no update on the memory usage of the Meteor build tool.  I think the real solution is just building a smaller app.  And that's one of the focuses of our upcoming performance work on Reaction.